### ISSUE TITLE
Why does "packed:true" increase the size than pacekd : false options?

### ISSUE THREAD

**Issue by pouu69** [evshiron/nwjs-builder-phoenix#43]
Why does "packed:true" increase the size than pacekd : false options?

if i packed: false

win x64 size : 176MB

but packed: true

win x64 size : 530MB

is it okay?

my package.json
````
{
  "name": "tyle-v1.1",
  "version": "1.1.0",
  "main": "index.html",
  "scripts": {
    "dist": "build --win --x64 --mirror https://dl.nwjs.io/ .",
    "start": "run --x86 --mirror https://dl.nwjs.io/ ."
  },
  "author": "tyle",
  "license": "ISC",
  "description": "tyle",
  "devDependencies": {
    "nw-builder": "^3.2.0",
    "nwjs-builder-phoenix": "^1.12.1"
  },
  "chromium-args": "--allow-running-insecure-content --ignore-certificate-errors",
  "window": {
    "toolbar": true,
    "width": 1200,
    "height": 750,
    "title": "tyle",
    "icon": "./resources/icons/512x512.png"
  },
  "webkit": {
    "plugin": true
  },
  "node-remote": "*://*.tyle.io",
  "user-agent": "tyle-nw-%ver",
  "inject-js-end": "./index.js",
  "dependencies": {
    "del": "^2.2.2",
    "fs": "0.0.1-security",
    "fs-jetpack": "^0.10.2",
    "ncp": "^2.0.0",
    "os": "^0.1.1",
    "path": "^0.12.7",
    "request": "^2.79.0",
    "semver": "^5.3.0",
    "url": "^0.11.0"
  },
  "build": {
    "nwVersion": "stable",
    "output": "./build/",
    "packed": true,
    "excludes": [
      "./node_modules/nwjs-builder-phoenix/**/*",
      "./node_modules/nw-builder/**/*"
    ],
    "mac": {
      "icon": "./resources/osx/tyle_logo_electron.icns",
      "copyright": "tyle.io"
    },
    "win": {
      "icon": "./resources/windows/tyle_logo_electron.ico"
    }
  }
}
````

**Comment by evshiron** [comment]
Because the generated file list includes directories, and when passed to `7zip-bin`, it take all files of these directories, as a result exclusion for files doesn't work when `packed: true`. Will fix.

**Comment by pouu69** [comment]
I look forward to fix soon.

i don't have time ㅜㅜ sorry

**Comment by evshiron** [comment]
Fixed in `nwjs-builder-phoenix` 1.13.1.

**Comment by pouu69** [comment]
thx working very well
